girder_annotation/girder_large_image_annotation/models/annotation.py

```diff
@@ -437,6 +437,59 @@ class AnnotationSchema:
         'description': 'An image overlay on top of the base resource.',
     })
 
+    pixelMapCategorySchema = {
+        'type': 'object',
+        'properties': {
+            'fillColor': colorSchema,
+            'strokeColor': colorSchema,
+            'label': {
+                'type': 'string',
+                'description': 'A string representing the semantic '
+                               'meaning of regions of the map with '
+                               'the corresponding color.'
+            },
+            'description': {
+                'type': 'string',
+                'description': 'A more detailed explanation of the '
+                               'meaining of this category.'
+            }
+        },
+        'required': ['fillColor']
+    }
+
+    pixelmapSchema = extendSchema(overlaySchema, {
+        'properties': {
+            'type': {
+                'type': 'string',
+                'enum': ['pixelmap'],
+            },
+            'values': {
+                'type': 'array',
+                'items': {'type': 'integer'},
+                'description': 'An array where the indices '
+                               'correspond to pixel values in the '
+                               'pixel map image and the values are '
+                               'used to look up the appropriate '
+                               'color in the categories property.'
+            },
+            'categories': {
+                'type': 'array',
+                'items': pixelMapCategorySchema,
+                'description': 'An array used to map between the '
+                               'values array and color values. '
+                               'Can also contain semantic '
+                               'information for color values.'
+            },
+            'boundaries': {
+                'type': 'boolean',
+                'description': 'True if the boundaries of the pixelmap regions have '
+                               'their own value in the data or segments array.'
+            },
+        },
+        'required': ['values', 'categories', 'boundaries'],
+        'description': 'A tiled pixelmap to overlay onto a base resource.'
```

Comment:
Please add `'additionalProperties': False` to this and the pixelmapCategorySchema.

Reply:
Added in 95bab98